api/structure/schema.py

```diff
@@ -0,0 +1,40 @@
+import graphene
+from graphql import GraphQLError
+from graphene_sqlalchemy import SQLAlchemyObjectType
+from sqlalchemy import func
+from helpers.auth.authentication import Auth
+from api.structure.models import Structure as StructureModel
+
+
+class Structure(SQLAlchemyObjectType):
+    class Meta:
+        model = StructureModel
+
+
+class Query(graphene.ObjectType):
+    """
+      Query for office structures
+    """
+    all_structures = graphene.List(Structure)
+    structure_by_web_id = graphene.Field(Structure, web_id=graphene.String())
+
+    @Auth.user_roles('Admin')
+    def resolve_all_structures(self, info):
+        """
+          Returns a list containing all office structures
+        """
+        query = Structure.get_query(info)
```

Comment:
@RachaelNantale  are you talking about this?

Reply:
`all_structures = graphene.List(Structure, description ="description goes here")`
Try something like this

Reply:
@RachaelNantale I have implemented the change